gstreamer-editing-services-sys/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[badges.travis-ci]
+branch = "master"
+repository = "sdroege/gstreamer-sys"
+
+[build-dependencies]
+pkg-config = "0.3.7"
+
+[dependencies]
+libc = "0.2"
+
+[dependencies.gio-sys]
+git = "https://github.com/gtk-rs/sys"
+
+[dependencies.glib-sys]
+git = "https://github.com/gtk-rs/sys"
+
+[dependencies.gobject-sys]
+git = "https://github.com/gtk-rs/sys"
+
+[dependencies.gstreamer-base-sys]
+path = "../gstreamer-base-sys"
+
+[dependencies.gstreamer-pbutils-sys]
+path = "../gstreamer-pbutils-sys"
+
+[dependencies.gstreamer-sys]
+path = "../gstreamer-sys"
+
+[dev-dependencies]
+shell-words = "0.1.0"
+tempdir = "0.3"
+
+[features]
+dox = []
+v1_16 = ["v1_8"]
+v1_6 = []
+v1_8 = ["v1_6"]
+
+[lib]
+name = "gstreamer_editing_services_sys"
+
+[package]
+authors = ["Thibault Saunier <tsaunier@igalia.com>"]
```

Comment:
Please also add me here so that once we have it on crates.io I can also upload new versions together with all the other crates :)

Reply:
Sure! Didn't know it work this way :-)